src/hooks/useOAuthFlow.tsx

```diff
@@ -39,9 +52,11 @@ const OAuthContext = createContext<OAuthConfig>({
 export const OAuthContextProvider = ({
   authConfig,
   children,
+  providedAuthorize = () => authorize(authConfig),
```

Comment:
You could consider hoisting this injected authorizer into the DeveloperConfig for a simple override without having to rebuild the default providers.

Reply:
looking briefly, if I am understanding, that looks great! that way it would be outside of the react context? yeah, we should definitely prefer doing that when possible cc @aecorredor

Reply:
And I think we might want something more like `methods: { authorize, refresh, revoke }`

Reply:
or maybe you mean just adding to `src/common/RootProviders.tsx`? I agree, we should do that so that users can continue using that root provider and not re-create their own